.jshintrc

```diff
@@ -29,5 +29,8 @@
     "maxparams": 3,
 
     // Limit the nesting to three levels for callbacks.
-    "maxdepth": 3
+    "maxdepth": 3,
+
+    // Identify possibly unsafe line breaks.
+    "laxbreak": true
```

Comment:
Just did some testing, `laxbreak` aims to check the unsafe line breaks, here is my testing case:

test.js
```js
function tester(){

    // Unsafe line break.
    return
    {
        UserName: 'Juwai'
    }
}
```
This function returns an undefined, not an Object as expected, I think it's an unsafe line break.
here is my .jshintrc file:

```
{
    "laxbreak": false
}
```
whether  `laxbreak` is `true` or `false` the jshint reports the same error.

Do you know an example about the `laxbreak` parameter?